### DISCUSSION TITLE
Run action from GitLab CI/CD Pipeline

### DISCUSSION THREAD

**Discussion by CarrotManMatt** [xalvarez/prevent-file-change-action#878] [Q&A] [answered]
Hi, I really appreciate this action being available to use on my Django projects hosted on GitHub, to prevent migrations from being deleted. Is there any way to carry this over to GitLab so it can be called from the command line within the GItLab Ci/CD pipeline runner?

**Comment by xalvarez** [comment, marked as answer]
Hi @CarrotManMatt,

Personally, as the maintainer, I don't have any experience with GitLab, and frankly, it wouldn't be high on my priority list to figure this out. I would prefer to leave this as it is, a GitHub action. That being said, this repository is open source and MIT-licensed, so if someone else wants to create a different repository to tackle the mentioned GitLab CLI issue, that would be absolutely fine, as long as the MIT license is respected.
